Expose onError, style props on underlying WebView

tylercote opened the pull request.

minond commented on src/sso/properties.ts:
```diff
@@ -20,6 +22,10 @@ export type ApiResponseBody = {
 }
 
+export type WebViewProps = {
+  onError?: (event: WebViewErrorEvent) => void;
```
How about we rename this to `onWebViewError`?

minond commented on src/sso/properties.ts:
```diff
@@ -1,10 +1,12 @@
+import { WebViewErrorEvent } from "react-native-webview/lib/WebViewTypes"
```
Question: can we import this from `react-native-webview` instead of `react-native-webview/lib/WebViewTypes`?

minond commented on src/components/renderer.tsx:
```diff
@@ -10,7 +10,8 @@ import { useFullscreenStyles } from "./screen_dimensions"
 
 export type StylingProps = {
-  style?: StyleProp<ViewStyle>
+  style?: StyleProp<ViewStyle>,
```
Nit pick: let's remove the comma after `style?: StyleProp<ViewStyle>`. They're optional in TS type defs and this will match what we're doing in other parts of the codebase.

minond commented on CHANGELOG.md:
```diff
@@ -7,6 +7,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [Unreleased]
 ### Added
+- Ability to style underlying WebView
+- Prop to handle errors on the WebView
```
Thanks for updating this!

minond reviewed (commented):
@tylercote this is looking awesome. I left some comments/feedback but there's nothing major.

tylercote commented:
Thanks for the feedback! I implemented everything.

VSCode was telling me that:
`import { WebViewErrorEvent } from "react-native-webview"`
But it still appears to work.

I also changed webviewStyle to webViewStyle just to match the case of onWebViewError.

minond reviewed (approved):
This looks good to me, thanks for doing this @tylercote !

minond commented:
@tylercote published to npm as v1.1.0, heads up

tylercote commented:
@minond Awesome! Thank you.
